src/common/grpc/src/channel_manager.rs

```diff
@@ -28,6 +28,7 @@ use tower::make::MakeConnection;
 use crate::error::{CreateChannelSnafu, InvalidConfigFilePathSnafu, InvalidTlsConfigSnafu, Result};
 
 const RECYCLE_CHANNEL_INTERVAL_SECS: u64 = 60;
+const DEFAULT_REQUEST_TIMEOUT_SECS: u64 = 2;
```

Comment:
Maybe too short, I think 5~10 seconds is more reasonable. Is it configurable?

Reply:
yes, set to 8 seconds when creating clients for datanodes in frontend